frontend/shared-components/enrollment/call-to-action/call-to-action.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import Link from 'next/link'
+import styles from './call-to-action.module.css'
+import { CTATypes, enrollmentCTAText } from '../../../models'
```

Comment:
this seems more like a sharable composition than a generaric component?

Reply:
I'm not familiar with that distinction! Do you mind elaborating?